test/basic.html

```diff
@@ -74,6 +74,22 @@
         }, 12);
       });
 
+      test('show() will accept only text and duration parameters', function() {
+        toast = fixture('basic');
+        toast.show({text: 'hello world', duration: 20});
+        assert.isTrue(toast.opened, '`opened` is true');
+        assert.equal(toast.text, 'hello world', '`text` is correct');
+        assert.equal(toast.duration, 20, '`duration` is correct');
+        toast.show({foo: 'bar'});
+        assert.isUndefined(toast.foo, '`foo` is not a valid propery and will not be set');
+      });
+
```

Comment:
Can you also add a test `show() does not accept protected and private properties` ?